models/user.go

```diff
@@ -100,9 +100,9 @@ func DeleteUser(ctx context.Context, u *user_model.User) (err error) {
 
 		// Delete Comments
 		const batchSize = 50
-		for start := 0; ; start += batchSize {
-			comments := make([]*issues_model.Comment, 0, batchSize)
-			if err = e.Where("type=? AND poster_id=?", issues_model.CommentTypeComment, u.ID).Limit(batchSize, start).Find(&comments); err != nil {
+		for {
+			comments := make([]*issues_model.Comment, 0, 50)
```

Comment:
This should still be batchSize

```suggestion
			comments := make([]*issues_model.Comment, 0, batchSize)
```